### ISSUE TITLE
Remove references to `fui-*` classes and IDs to link JS behaviour

### ISSUE THREAD

**Issue by engram-design** [verbb/formie#929]
Currently, we're pretty heavily tied to using classes and IDs for JS functionality, which we should change to `data-*` attributes for greater flexibility with templating.

Whether we also move the base CSS to use data attributes in another consideration, but likely to leave as-is, and separate style vs behaviour.

**Comment by engram-design** [comment]
Added for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0](https://github.com/verbb/formie/releases/tag/2.0.0)
